### DISCUSSION TITLE
Elastalert failed to sync

### DISCUSSION THREAD

**Discussion by ranimhassine** [Security-Onion-Solutions/securityonion#13506] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

32

### Storage for /

.

### Storage for /nsm

140

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

![image_2024-08-16_103040871](https://github.com/user-attachments/assets/5adce554-33bc-4813-ac1b-73e16ce4202a)
how can i change the elastalert status to ok because the soc is not alerting

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
If you click on the words "Sync Failed" it should bring up a Hunt window with the relevant logs. Are there any issues in there?
